indexer/osm_editor.hpp

```diff
@@ -154,6 +154,10 @@ class Editor final
     string m_uploadStatus;
     string m_uploadError;
   };
+  /// @returns pointer to m_features[id][index] if exists, nullptr otherwise.
+  FeatureTypeInfo const *
```

Comment:
С трудом читается.

> On 14.4.2016, at 16.24, mgsergio <notifications@github.com> wrote:
> 
> In indexer/osm_editor.hpp:
> 
> > @@ -154,6 +154,10 @@ class Editor final
> >      string m_uploadStatus;
> >      string m_uploadError;
> >    };
> > +  /// @returns pointer to m_features[id][index] if exists, nullptr otherwise.
> > +  FeatureTypeInfo const *
> Норамальный перенос. Стандартная для с++ штука.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>